Zishy.yml

```diff
@@ -0,0 +1,53 @@
+name: Zishy
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - zishy.com/albums
+    scraper: sceneScraper
+
+galleryByURL:
+  - action: scrapeXPath
+    url:
+      - zishy.com/albums
+    scraper: galleryScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: &title
+        selector: //head/title
+        postProcess:
+          - replace:
+            - regex: ([\w ]*) - Zishy
+              with: $1
+      Details: &details
+        selector: //div[@id='descrip']
+      Date: &date
+        selector: //div[@id='headline']/span[last()]
+        postProcess:
+          - replace:
+            - regex: added on (\w{3}) (\d{2}). (\d{4})
+              with: $3-$1-$2
+          - parseDate: 2006-Jan-02
+      Image:
+        selector: //video[@poster]/@poster
+        postProcess:
+          - replace:
+            - regex: /(.*)
+              with: https://zishy.com/$1
```

Comment:
Do you have any urls available where images match this?
I tried the galleries with video part but the few i tried had the image of the video in the `Bonus HD Video Clip` part
eg
`https://www.zishy.com/albums/2077-ariel-gossimer-too-much-awesome-2`
To get the images from those something like
```yaml
      Image:
        selector: //div[@class="player"]//@style[contains(.,"background")]
        postProcess:
          - replace:
            - regex: .+url\((/uploads/.+)\);.+
              with: https://zishy.com/$1
```
works for me

Reply:
Thank you for that. Your change works much better and grabs the actual poster frame of the video from the page. I have made the adjustments.